### PR TITLE
Styling fixes for email subscription page

### PR DESCRIPTION
**Pull request by NetOperatorWibby**
Builds upon/closes https://github.com/lbryio/lbry.io/issues/723.

Also in this PR is a fix for not having an API server to fall back on. This is tested and works. If there's an API server listed in the config file, it is used. If not, `api.lbry.io` is used.

### CODE REVIEW

File: lib/thirdparty/LBRY.class.php [hunk 1]
```diff
@@ -3,9 +3,12 @@
 
 class LBRY
 {
-    public static function getApiUrl($endpoint)
-    {
+    public static function getApiUrl($endpoint) {
+      if (strlen(Config::get(Config::LBRY_API_SERVER)) > 0) {
```
[the remark this answers is not in the document]
Comment (NetOperatorWibby) [reply lib/thirdparty/LBRY.class.php]: Ah okay. I don't know PHP, I came up with this after 10 minutes of Internet searching.

**Comment by tzarebczan** [comment]
@NetOperatorWibby / @kauffj what's the next step here?

**Comment by NetOperatorWibby** [comment]
@kauffj Made requested changes

**Comment by tiger5226** [comment]
I did run it and it still works. Below is a screenshot:

screen shot 2018-09-20 at 8 36 50 pm

**Comment by tiger5226** [comment]
One thing that I would add is how we handle commits. rebasing is preferred vs merging to keep commits organized properly. Technically, you should just start off form master, then rebase before merging into master.

**Comment by NetOperatorWibby** [comment]
I've rebased and squashed all commits into one.

File: lib/thirdparty/LBRY.class.php [hunk 2]
```diff
@@ -5,6 +5,10 @@ class LBRY
 {
     public static function getApiUrl($endpoint)
     {
+        if (!strlen(Config::get(Config::LBRY_API_SERVER)) > 0) {
+            throw new Exception("API server URL is missing from configuration");
```
Comment (kauffj) [inline lib/thirdparty/LBRY.class.php]: @NetOperatorWibby This might seem small, but even referencing the config path creates a difficult to trace dependency (if someone was refactoring configuration logic, they shouldn't be expected to update this unrelated file if the API signature of `Config` didn't change).

File: view/template/mail/_settingsForm.php [hunk 3]
```diff
@@ -7,12 +7,19 @@
...
+    <?php if ($error): ?>
+      <div class="notice notice-error spacer1"><?php echo $error ?></div>
+    <?php endif ?>
+    <div class="notice notice-success hide">Your email preferences have been updated.</div>
+  </div>
+
+
   <section class="email-section">
-    <h4>Where do you want to receive email?</h4>
+    <h4><?php echo count($emails ) > 1 ? 'Receiving Addresses' : 'Do You Want To Receive Mail?' ?></h4>
```
Comment (kauffj) [inline view/template/mail/_settingsForm.php]: I think part of the right answer to reducing confusion here is to change the behavior for 1 vs. many emails

File: view/template/mail/_settingsForm.php [hunk 4]
```diff
@@ -7,12 +7,19 @@
   Javascript is required to securely send your unsubscribe information. Email <a href="mailto:help@lbry.io" class="link-primary">help@lbry.io</a> for manual unsubscription.
 </noscript>
 <form id="<?php echo $formId ?>" novalidate style="display: none">
-  <?php if ($error): ?>
-    <div class="notice notice-error spacer1"><?php echo $error ?></div>
-  <?php endif ?>
-  <div class="notice notice-success hide">Your email preferences have been updated.</div>
+  <div style="min-height: 48px">
```
Comment (kauffj) [inline view/template/mail/_settingsForm.php]: I made the form auto-save. This is an awful design from basically every standard (code quality, maintainability, UX), but otherwise the form would jump on save and the only other alternative was to bite bullet and add snack bars or equivalent UI element.

**Comment by kauffj** [comment]
Hmm, I submitted a version that autosaved and left comments on it, but there's now a bug if you change things too fast. Refactoring again...

**Comment by kauffj** [comment]
OK, I think this version is good. @NetOperatorWibby please review and merge if you like it from a UI/UX perspective.

My biggest complaint is jumping on save which is a big UX no-no, but I don't see a way to avoid it without adding a new component styling like a snack bar.

**Comment by kauffj** [comment]
I can't seem to assign this PR to @NetOperatorWibby  - @alyssaoc can you look into if he is missing from a proper group or something?

**Comment by kauffj** [comment]
Sorry - not assign - request as reviewer. He is assigned.

**Comment by tzarebczan** [comment]
Probably can't add as a reviewer since he opened the PR.
